packages/engine/Source/Scene/ScreenSpaceCameraController.js

```diff
@@ -271,6 +271,11 @@ function ScreenSpaceCameraController(scene) {
    * @default true
    */
   this.enableCollisionDetection = true;
+  /**
+   * If set, the camera will not be able to tilt past this angle, expressed in radians.
+   * @type {number}
+   */
```

Comment:
```suggestion 

 * @example
 * // prevent camera from tilting below ellipsoid surface
 * controller.maximumTiltAngle = Math.PI / 2
   */

Reply:
Thanks for the suggestion. This feels like a good thing to add so I have pushed this update to this MR